### PR TITLE
Rename Activity Manager into Teamwork Assistant

### PR DESCRIPTION
**Pull request by fred2gombert**

### CODE REVIEW

File: cookbook/activity_manager/index.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-How To Customize Activity Manager (Enterprise Edition)
```
Comment (pierallard) [inline cookbook/activity_manager/index.rst]: please rename folder too (it will be visible on the URL)

**Comment by pierallard** [comment]
Apparently @fred2gombert can not rename folders, so don't forget this issue after merge please
https://github.com/akeneo/pim-docs/issues/484

**Comment by mickaelandrieu** [comment]
@pierallard I'll finish this one, thank for notice the issue :)

**Comment by mickaelandrieu** [comment]
Rebased and folder renamed => GTM

**Pull request merged**